obsplus/utils/stations.py

```diff
@@ -114,19 +164,74 @@ def _get_resp_key(key):
             return tuple(key)
 
     def _maybe_add_response(series, channel_kwargs):
-        """ Maybe add the response information if required columns exist. """
+        """ Simple dispatch to determine which response function to use."""
+        # if no response columns are found exit early
+        if response_cols.isdisjoint(set(series.index)):
+            return
+        _maybe_add_nrl_response(series, channel_kwargs)
+        if "response" not in channel_kwargs:
```

Comment:
Am I understanding correctly that "response" gets added if it succeeds in fetching the NRL response?